app/models/feedback.rb

```diff
@@ -60,13 +72,13 @@ def save_feedback
     response[:success]
   end
 
-  def save_bug_report
+  def send_to_zendesk(success, failure, log_message)
     ZendeskSender.send!(self)
-    @response_message = 'Fault reported'
+    @response_message = success
   rescue ZendeskAPI::Error::ClientError => e
-    @response_message = 'Unable to submit fault report'
+    @response_message = failure
     LogStuff.error(class: self.class.name, action: 'save', error_class: e.class.name, error: e.to_s) do
-      'Bug report submisson failed!'
+      log_message
     end
     false
   end
```

Comment:
that's clever. yep that works thank you

Reply:
updated it :) . I used `feedback_type` as the parameter name only because `type` is an existing attr_accessor / variable name

Reply:
Ah, I wonder if we can use that `type` variable to simplify things more? So instead of passing in the type, we can just do 

```
  def send_to_zendesk
    feedback_type = type.humanize
    ZendeskSender.send!(self)
    @response_message = success
    @response_message = "#{feedback_type} submitted"
  rescue ZendeskAPI::Error::ClientError => e
    @response_message = failure
    @response_message = "Unable to submit #{feedback_type.downcase}"
    LogStuff.error(class: self.class.name, action: 'save', error_class: e.class.name, error: e.to_s) do
      log_message
      "#{feedback_type} submisson failed!"
    end
    false
  end
```

and then we don't need to pass in any arguments at all when we call the method